### PR TITLE
test: Session saving and loading with the fileStore

### PR DESCRIPTION
**Pull request by ffflorian**
This PR adds a test case for #98.

### CODE REVIEW

File: test/node/store/CryptoboxCRUDStoreSpec.js [hunk 1]
```diff
@@ -152,4 +153,41 @@ describe('cryptobox.store.CryptoboxCRUDStore', () => {
...
+          return alice.load_session_from_cache(sessionId);
+        })
+        .then(cryptoboxSession => {
+          expect(cryptoboxSession.fingerprint_remote()).toBe(bob.public_key.fingerprint());
+          // Check if code is robust and can handle a second call
+          return alice.session_from_prekey(sessionId, bobPreKeyBundle.serialise());
+        })
+        .then(cryptoboxSession => {
+          expect(cryptoboxSession.fingerprint_remote()).toBe(bob.public_key.fingerprint());
+          // Remove local cache and reinforce a session from PreKey
```
Comment (bennyn) [inline test/node/store/CryptoboxCRUDStoreSpec.js]: `// Remove local cache and reinforce a session from PreKey` -> `// Remove local cache and reinforce a session load from store`

File: test/node/store/CryptoboxCRUDStoreSpec.js [hunk 2]
```diff
@@ -152,4 +153,41 @@ describe('cryptobox.store.CryptoboxCRUDStore', () => {
         .catch((error) => done.fail(error));
     });
   });
+
+  describe('session_from_prekey', function() {
+    fit('saves and caches a valid session from a serialized PreKey bundle', function(done) {
```
Comment (bennyn) [inline test/node/store/CryptoboxCRUDStoreSpec.js]: `fit` -> `it`

Comment (bennyn) [reply test/node/store/CryptoboxCRUDStoreSpec.js]: Maybe it also good to have one test case for "saves and caches a valid session from a serialized PreKey bundle" and another one for "loads session from database when it's not available in cache" to make it clear which specific part is tested.

Comment (ffflorian) [reply test/node/store/CryptoboxCRUDStoreSpec.js]: Thanks, will do

**Pull request merged**